src/main/java/mate/academy/service/imp/MovieServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.service.imp;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) {
+        return movieDao.get(id).get();
```

Comment:
Potentially, you can get a NoSuchElementException here. In my opinion you should handle this in MovieDaoImpl.get()